docs/support/troubleshooting.md

```diff
@@ -49,60 +48,116 @@ To resolve this, try either one of the following methods:
 
 !!! warning
 
-    Note that some components in `autoware.universe` don't work without CUDA.
+    Note that some components in Autoware Universe require CUDA, and only CUDA 11.4 is supported at this time.
```

Comment:
Mm, if possible, I don't want to write the version here because we'll update it soon.
Considering the maintenance cost, replacing this with a link is better.

Reply:
Like this?

```suggestion
    Note that some components in Autoware Universe require CUDA, and only the CUDA version in the [env file](https://github.com/autowarefoundation/autoware/blob/main/amd64.env) is supported at this time.
```

Reply:
That fixes the problem perfectly!

I originally said something like `only the CUDA version stated in the installation guide is supported`, but thought that the version is only mentioned in the manual installation steps and that would also change in the future so decided to write it here instead, which just moved the problem to a different place. 😅